python-package-helpers.cmake

```diff
@@ -20,12 +20,22 @@ function (ev_create_pip_install_dist_target)
         set(EV_CREATE_PIP_INSTALL_DIST_TARGET_PACKAGE_SOURCE_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
     endif()
 
-    set(CHECK_DONE_FILE "${CMAKE_BINARY_DIR}/${EV_CREATE_PIP_INSTALL_DIST_TARGET_PACKAGE_NAME}_pip_install_dist_installed")
+    execute_process(
+        COMMAND
+            ${Python3_EXECUTABLE} -c "from setuptools import setup; setup()" --version
```

Comment:
Is this compatible with packages that are without `setup.py`  file?

Reply:
It works with ev-cli which only uses a setup.cfg and pyproject.toml, but probably depends on your setuptools version

Reply:
Tested it, works also for `pyproject.toml` only ✅